python-sdk/tests_integration/test_example_dags.py

```diff
@@ -83,15 +85,14 @@ def order(dag_id: str) -> int:
     return -1
 
 
-dag_bag = get_dag_bag()
+@pytest.fixture()
+def example_dag_ids():
+    dag_bag: DagBag = get_dag_bag()
+    for dag_id in sorted(dag_bag.dag_ids, key=order):
+        yield dag_id, dag_bag
 
 
-@pytest.mark.parametrize("dag_id", sorted(dag_bag.dag_ids, key=order))
-def test_example_dag(session, dag_id: str):
+def test_example_dag(session, example_dag_ids):
+    dag_id, dag_bag = example_dag_ids
```

Comment:
Would it make sense to assert dag_bag import_errors is false?

Reply:
@tatiana We are doing this at the time of creation in the `get_dag_bag()` method.